src/hackney_http.erl

```diff
@@ -325,7 +328,40 @@ parse_trailers(St, Acc) ->
     _ -> error
   end.
 
-parse_body(St=#hparser{body_state=waiting, te=TE, clen=Length,
+-define(MAX_WBITS, 15).	% zconf.h
+parse_body(St=#hparser{body_state=waiting, ce=CE, clen=Length, method=Method}) ->
+  St2 = case CE of
```

Comment:
I think it would be nice to use `parse_body` function clauses instead of this case

Reply:
Maybe, but I was following style hints from @benoitc in the handling of `te` which is handled with a `case`.  Personally I am not bothered.

Reply:
updated